.github/workflows/test-workflow.yaml

```diff
@@ -0,0 +1,31 @@
+name: test-workflow
+on: [push]
+jobs:
+  test-darwin:
+    runs-on: macos-10.15
```

Comment:
Why do we need two darwin jobs?

Reply:
There were two builds happening, one on CircleCI one on buildkite, difference between them is being preserved by splitting those into two different jobs, as I was unsure if I should merge them.

Reply:
That makes sense to me. But the two new jobs run on macos right? While the buildkite job runs on linux.

Reply:
Also, I think it would be ok to put both jobs in a single workflow and have a single configuration file, unless you need them split for some reason.